dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -266,14 +266,10 @@ public boolean canPublishDataverse() {
         return permissionsWrapper.canIssuePublishDataverseCommand(dataset.getOwner());
     }
 
-    public boolean latestDatasetHasFiles(){
+    public boolean isLatestDatasetHasFiles(){
```

Comment:
Much better, however now it looks a bit strange, maybe something like
`isLatestDatasetWith(Any/-)Files(Included/Added/Uploaded/-)`?